### PR TITLE
paint_board-10: Moved color panel to independent component

### PR DESCRIPTION
**Pull request by KoTuK2306**

### CODE REVIEW

File: src/components/color-panel/color-panel.style.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+import { Colors } from '../../constants/colors';
+
+export const ColorsStyles = createUseStyles({
+  color: {
+    height: 14,
+    width: 14,
+    border: `1px solid ${Colors.black}`,
+    borderRadius: 6,
+    background: ({ color }) => color,
```
Comment (pavelsokolov21) [inline src/components/color-panel/color-panel.style.ts]: `background: ({ color }: Record<string, string>) => color`

File: src/components/color-panel/index.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export { ColorPanel } from './color-panel';
```
Comment (pavelsokolov21) [inline src/components/color-panel/index.ts]: if color panel will be only in header component, then you can put this component in the header folder

**Review by pavelsokolov21 — CHANGES_REQUESTED** [review]

File: src/components/index.ts [hunk 3]
```diff
@@ -1 +1,2 @@
 export { Header } from './header';
+export { ColorPanel } from './header/color-panel';
```
Comment (pavelsokolov21) [inline src/components/index.ts]: It is not necessary because this component is used only in header

**Review by pavelsokolov21 — CHANGES_REQUESTED** [review]

**Review by pavelsokolov21 — APPROVED** [review]

File: src/components/header/color-panel/color-panel.tsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import { FC } from 'react';
+import classnames from 'classnames';
+
+import { Styles, ColorsStyles } from './color-panel.style';
+import { Colors } from '../../../constants';
+
+export const ColorPanel: FC = () => {
```
Comment (IliaSolovev) [inline src/components/header/color-panel/color-panel.tsx]: I think that its panel has more than 1 color. But when I read this name I don`t think so

**Review by IliaSolovev — CHANGES_REQUESTED** [review]

**Review by pavelsokolov21 — APPROVED** [review]

**Review by IliaSolovev — APPROVED** [review]

**Pull request merged**